requirements.txt

```diff
@@ -9,4 +9,5 @@ pandas>=0.25.0
 ibmcloudsql>=0.3.6
 flatten_json==0.1.7
 adal>=1.2.2
+flask>=1.1.1
```

Comment:
Flask is only used by the proxy connector which is meant to be run by a developer in their dev environment. This may need to be removed due to the refactor work currently underway for stix-shifter, just wondering why you included the import here.

Reply:
As you rightly pointed out its not needed. Removed it, its already there in requirements_dev.txt.